src/components/Modals/UseAsAliasModal/UseAsAliasModal.container.ts

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import { RootState } from 'modules/common/types'
+import { getENSUsedAsAlias, getLoading } from 'modules/ens/selectors'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './UseAsAliasModal.types'
+import UseAsAliasModal from './UseAsAliasModal'
+import { setAliasRequest, SET_ALIAS_REQUEST } from 'modules/ens/actions'
+import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
```

Comment:
Absolute imports to the top! remember the [standards](https://github.com/decentraland/standards/blob/master/standards/react-redux.md#5-imports-order-on-files)

Reply:
btw, Did your linter fixes take care of this type of errors? not mine.